docs/_rtd/virtualenv.md

```diff
@@ -1,3 +1,31 @@
 ## Setting Up Your Virtual Environment
 
-Add information about how to set up Anaconda.
+If you wish to do higher-level development on top of `ark`, we recommend setting up a virtual environment. We highly recommend using `conda` virtual environments. To be able to set one up, you will need to install the Anaconda package.
+
+### Installing Anaconda
+
+For a step-by-step guide of how to install Anaconda, please refer to these links:
+* https://docs.anaconda.com/anaconda/install/mac-os/ for Mac users
+* https://docs.anaconda.com/anaconda/install/windows/ for Windows users
+
+We recommend using the graphical installer for ease of use.
+
```

Comment:
should we mention miniconda at all?  It's definitely an easier/lighter install.

Reply:
@ackagel I didn't because I think most people who will be developing on top of `ark-analysis` or even just using `ark-analysis` functions for their own development should be fine with a heavier installation. Just a personal opinion though.